### DISCUSSION TITLE
Keyball44 キーソケット接点剥がれの回避策

### DISCUSSION THREAD

**Discussion by boxwood8** [Yowkees/keyball#383] [Q&A] [answered]
![image](https://github.com/Yowkees/keyball/assets/147312996/82449b85-ff58-44f6-83af-795723e6a74e)

キーソケットの方向を修正しようとして赤X部の接点を切ってしまいました。
現在赤枠のスイッチが反応しないのですが、緑と青同士をワイヤーで直結すれば良いでしょうか。

**Comment by Yowkees** [comment, marked as answer]
ご連絡ありがとうございます。

緑どうしは合っていますが、青が違います。
緑どうしをワイヤーで接続後、下記画像のオレンジどうしを接続してください。
これでキーを復活できると思います。

以上、お手数ですがご対応お願いいたします。

![KB44_col3fix](https://github.com/Yowkees/keyball/assets/84209772/ea248712-a05b-427c-bc5d-452083bd409b)

**Reply by boxwood8** [reply]
お忙しいなかご対応ありがとうございました！
早速試してみたところ、キーが認識されました！
助かりましたー

**Comment by Yowkees** [comment]
無事にキー入力ができるようになったようで安心しました。
ぜひご愛用いただければ幸いです。
